package.json

```diff
@@ -8,8 +8,24 @@
     "start": "next start"
   },
   "dependencies": {
+    "isomorphic-unfetch": "^3.0.0",
     "next": "9.1.7",
     "react": "16.12.0",
     "react-dom": "16.12.0"
+  },
+  "prettier": {
+    "singleQuote": false,
+    "semi": true
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
+    }
+  },
+  "lint-staged": {
+    "*.js": [
+      "prettier --write",
+      "git add"
+    ]
```

Comment:
Husky és lint-staged működik akkor, ha a dev környezet dockerben fut, de a host gépről commitolok?

Reply:
Nem, ezek git hook-ok, csak akkor működnek, ha a git parancsokat is ugyanott használod.

Reply:
Fun fact: a VSCode prettier extension felismeri a package.jsonben leírt prettier szabályokat szóval az maradhat